Add missing text issue #3539 (#3544)
OSGeo/gdal: gdal/doc/source/api/gdalogrin_java.rst

.. _gdalogrin_java:

================================================================================
GDAL/OGR In Java
================================================================================

> The GDAL project has SWIG generated Java bindings for GDAL and OGR. The Java bindings have been substantially reworked for 1.7.0 release. For the 1.5.0 and 1.6.0 releases, they are considered to be broken.

Reviewer: ```suggestion
The GDAL project has SWIG generated Java bindings for GDAL and OGR.
```

Generally speaking the classes and methods mostly match those of the GDAL and OGR C++ classes. You can find the `​Javadoc <http://gdal.org/java>`__ of the API of the Java bindings for
GDAL 1.7.0 and later releases.

Due to the fact the Java garbage collector works in a separate thread from the main thread, it is necessary to configure GDAL with multi-threading support,
even if you do not use GDAL API from several Java threads.

> The minimum version of Java required to build the Java bindings is Java 1.4.

Reviewer: ```suggestion
The minimum version of Java required to build the Java bindings is Java 7

```

Maven Users
-----------

> As of release 1.11.2 the Java bindings are available from the ​`Maven Central <http://search.maven.org/>`__ repository. All that is needed is to declare a dependency.

Reviewer: ```suggestion
The Java bindings are available from the ​`Maven Central <http://search.maven.org/>`__ repository. All that is needed is to declare a dependency.

```
